workflow-pipes.qmd

```diff
@@ -35,32 +35,33 @@ For example, the last chapter finished with a moderately complex pipe:
 
 flights |>  
   filter(!is.na(arr_delay), !is.na(tailnum)) |> 
-  group_by(tailnum) |> 
   summarise(
+    .by = tailnum,
     delay = mean(arr_delay, na.rm = TRUE),
     n = n()
-  )
+  ) |> 
+  filter(n > 5)
 ```
 
-Even though this pipe has four steps, it's easy to skim because the verbs come at the start of each line: start with the `flights` data, then filter, then group, then summarize.
+Even though this pipe has thre steps, it's easy to skim because the verbs come at the start of each line: start with the `flights` data, then filter, then group, then summarize.
```

Comment:
```suggestion
Even though this pipe has three steps, it's easy to skim because the verbs come at the start of each line: start with the `flights` data, then filter, then group, then summarize.
```